ui/src/components/home/NewHereSection.js

```diff
@@ -0,0 +1,141 @@
+
+  const resetForm = () => {
+    setName("");
+    setEmail("");
+    setNotes("");
+    setIsBot(true);
+    setSubmitted(false);
+  };
+
+  return (
+    <>
+      <Flex
+        w="full"
+        h={["100vh", "100vh", "100vh", "100vh", "62vh"]}
+        bgImage={`linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ),url(${newHereUrl})`}
+        bgSize="cover"
+        bgPosition="center center"
+        justify="center"
+      >
+        <Flex
+          w="full"
+          h="100%"
+          justify="center"
+          background='linear-gradient(90deg, rgba(6, 40, 163, 0.6) 0%, rgba(196, 241, 249, 0.6) 100%);"    '
+          style={{ backdropFilter: "blur(7px)" }}
+        >
+          <Container maxW="container.lg" justifyContent="center" display="flex">
+            <VStack color="white" justify="center" spacing={6}>
+              <Heading fontSize={["1.3em", "3em"]}>
+                New here? Connect with us
+              </Heading>
+              <form onSubmit={onSubmit}>
+                <VStack spacing={[8, 4]} alignItems="center">
+                  <FormControl id="name" isRequired w={["85%", "100%"]}>
+                    <Input
+                      variant="flushed"
+                      type="name"
+                      placeholder="Name"
+                      onChange={(e) => setName(e.target.value)}
+                    />
+                  </FormControl>
+                  <FormControl id="email" isRequired w={["85%", "100%"]}>
+                    <Input
+                      variant="flushed"
+                      type="email"
+                      placeholder="Email"
+                      onChange={(e) => setEmail(e.target.value)}
+                    />
+                  </FormControl>
+                  <FormControl id="notes" w={["85%", "100%"]}>
+                    <Input
+                      type="notes"
+                      placeholder="Notes and remarks (Optional)"
+                      variant="flushed"
+                      onChange={(e) => setNotes(e.target.value)}
+                      _autofill={{
+                        background: "rgba(0,0,0,0)",
+                      }}
+                    />
+                  </FormControl>
+                  <Box transform={["scale(0.77)", "scale(1)"]}>
+                    <ReCAPTCHA
+                      sitekey={process.env.REACT_APP_CAPTCHA}
+                      onChange={onChange}
+                    />
+                  </Box>
+                  <CustomButton
+                    bg="rgb(0, 0, 0, 0)"
+                    variant="outline"
+                    color="white"
+                    _hover={{
+                      bg: "white",
+                      color: "#1A365D",
+                      borderColor: "#1A365D",
+                    }}
+                    text="Connect with us >"
+                    type="submit"
+                  />
+                  <FormLabel visibility={submitted ? "visible" : "hidden"}>
```

Comment:
is there a way to use `display: none` or something for this? kinda weird that it's just taking up space in the UI but maybe moving things around might make things weirder. or would a popup on the popup make more sense?

Reply:
using `display: none` moves the entire set of inputs up and down.